SplunkAppForWazuh/appserver/static/js/controllers/agents/agents/agents.html

```diff
@@ -103,6 +103,17 @@
                 </p>
               </div>
             </div>
+            <div class="euiFlexItem euiFlexItem--flexGrowZero">
+              <div class="euiStat">
+                <div class="euiText euiText--small euiStat__description">
+                  <p>Agents synced</p>
```

Comment:
Change to "Synced agents", as done in [Kibana](https://github.com/wazuh/wazuh-kibana-app/pull/3874/files#diff-9f89d90556bc520d79685e9c92d6d0452a844a9af08c5aba988bd291a6f69462R226)